### PR TITLE
add about db-type=kafka

### PR DESCRIPTION
**Pull request by july2993**
as title
when dowload from http://download.pingcap.org/tidb-binlog-latest-linux-amd64.tar.gz, it only contains the binary, i think it may be better if it can include the default config template file?

### CODE REVIEW

**Comment by GregoryIan** [comment]
@buggithubs PTAL

**Comment by GregoryIan** [comment]
Did you add anything to conflence？ @july2993

File: tools/tidb-binlog-kafka.md [hunk 1]
```diff
@@ -89,6 +91,20 @@ cd tidb-binlog-latest-linux-amd64
     dir = "/path/pb-dir"
     ```
 
+* Drainer 输出为 kafka，要在配置文件中设置如下参数：
+
+    ```
+    [syncer]
+    db-type = "kafka"
+
+    # when db-type is kafka, you can uncomment this to config the down stream kafka, it will be the globle config kafka default
```
Comment (GregoryIan) [inline tools/tidb-binlog-kafka.md]: what meaning of `It will be the global config kafka default`?

**Comment by july2993** [comment]
> Did you add anything to conflence？ @july2993
no

**Pull request merged**